docs/conf.py

```diff
@@ -28,6 +28,13 @@
     'stdlib': ('https://docs.python.org/3/library/%s.html', ''),
 }
 
+linkcheck_ignore = [
+    r'https://github.com/beetbox/beets/issues/',
+    r'https://github.com/\w+$',  # ignore user pages
```

Comment:
Seems like a good idea. In which case, maybe we should undo the one remaining change to the `:user:` links in the changelog?

Reply:
Well, it does lead to a broken user page. The motivation for ignoring user and issue links is that they come up so often and end up timing out the github connection altogether as we get rate limited.

Reply:
Legit, that makes sense.